src/rdkafka.c

```diff
@@ -1298,17 +1298,17 @@ static void rd_kafka_term_sig_handler (int sig) {
 
 
 rd_kafka_t *rd_kafka_new (rd_kafka_type_t type, rd_kafka_conf_t *app_conf,
-			  char *errstr, size_t errstr_size) {
```

Comment:
No whitespace only diffs please.

Reply:
One comment on this, the actual source code is riddled with a mix of leading tabs and whitespaces and often in the same function. I guess whomever made the changes a long time ago was not as careful. Some of the indentation i fixed was around my changes to match proper alignment with what i'm submitting. Otherwise it's impossible to make the code look good.

Reply:
Yes, this is unfortunate, but we're going to fix this throughout the code in one single commit when we have a style checker in place, before that I urge people not to commit whitespace diffs unless they change the line.

Reply:
cool